content/Hardware/Generic/Use-an-LCD-with-Arduino.md

```diff
@@ -0,0 +1,9 @@
+---
+title: "Use an LCD with Arduino"
+---
+
+You can easily connect a liquid crystal display (LCD) with an Arduino to display data.
+
+The [LiquidCrystal library](https://www.arduino.cc/reference/en/libraries/liquidcrystal/) allows you to control LCD displays based on the Hitachi HD44780 chipset (which is found on most text-based LCDs).
+
+* See [this guide](https://docs.arduino.cc/learn/electronics/lcd-displays) on how to wire an LCD to an Arduino board and use the LiquidCrystal library
```

Comment:
```suggestion
See [this guide](https://docs.arduino.cc/learn/electronics/lcd-displays) on how to wire an LCD to an Arduino board and use the LiquidCrystal library
```
I would remove the bullet since there's only one item in the "list".